src/helpers/spaces.ts

```diff
@@ -250,19 +258,22 @@ async function getFollowers(): Promise<
 }
 
 async function loadSpacesMetrics() {
-  const metricsFn = [getFollowers, getProposals, getVotes];
-  const results = await Promise.all(metricsFn.map(fn => fn()));
-
-  metricsFn.forEach((metricFn, i) => {
-    for (const [space, metrics] of Object.entries(results[i])) {
-      if (!spacesMetadata[space]) continue;
-
-      spacesMetadata[space].counts = {
-        ...spacesMetadata[space].counts,
-        ...metrics
-      };
-    }
-    log.info(`[spaces] ${metricFn.name.replace('get', '')} metrics loaded`);
+  const results = await Promise.all([
+    getFollowers(),
+    getProposals(),
+    getVotes()
+  ]);
+  const [followerMetrics, proposalMetrics, voteMetrics] = results;
```

Comment:
We should add new line if there is `await` call
```suggestion
  ]);
  
  const [followerMetrics, proposalMetrics, voteMetrics] = results;
```

Reply:
we have this in many places, better to add a eslint rule